docs/_posts/examples/3400-01-10-live-geojson.html

```diff
@@ -21,11 +21,10 @@
         map.getSource('drone').setData(url);
     }, 2000);
 
-    map.addSource('drone', { type: 'geojson', data: url });
```

Comment:
This example uses `map.getSource(...)`. I think it's clearer with the explicitly parallel `addSource`.